### PR TITLE
NO-ISSUE: Change quay ocpmetal to edge-infrastructure

### PR DESCRIPTION
**Pull request by mkowalski**
quay.io/ocpmetal repository has been deprecated for quite a while in
favour of quay.io/edge-infrastructure. This PR changes all the
references to the old one.

## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Assignees

/cc @osherdp 
/cc @mresvanis 

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: deploy/wiremock/wiremock-deployment.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
         - name: wiremock
-          image: quay.io/ocpmetal/wiremock
+          image: quay.io/edge-infrastructure/wiremock
```
Comment (osherdp) [inline deploy/wiremock/wiremock-deployment.yaml:18]: Also shouldn't exist

File: deploy/postgres/postgres-deployment-ephemeral.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
         - name: postgres
-          image: quay.io/ocpmetal/postgresql-12-centos7
+          image: quay.io/edge-infrastructure/postgresql-12-centos7
```
Comment (osherdp) [inline deploy/postgres/postgres-deployment-ephemeral.yaml:18]:
```suggestion
          image: quay.io/centos7/postgresql-12-centos7
```

**Review by osherdp — CHANGES_REQUESTED** [review]
be aware that prow will not take into consideration Dockerfile changes in this PR (but will take them from master)
so additional testing is needed in addition to CI

**Comment by mkowalski** [comment]
/close

Not relevant anymore

**Pull request closed without merging**